### ISSUE TITLE
Add-on inputstream.rtmp'

### ISSUE THREAD

**Issue by ryanfantus** [notspiff/inputstream.rtmp#38]

**Comment by mrplow87** [comment]
Yes, a version bump of 2.0.6 to 2.0.7 in line 9 of ```../addons/inputstream.rtmp/addon.xml``` fixes the compatibility issue in the nightly ```2:18.0+git20180709.0202-2f216c5-0bionic```.

Fixed line 9: ```<import addon="kodi.binary.instance.inputstream" version="2.0.7"/></requires>```

**Comment by mrplow87** [comment]
True. I'm slowly getting to know how everything is done with the packaging, so I didn't know that (yet) :)

**Comment by wsnipex** [comment]
since addon version was not bumped, packaging fails. Launchpad requires uploaded package version to be higher then the existing one

**Comment by wsnipex** [comment]
I've worked around this by increasing the patch version, new builds pending.
